index.html

```diff
@@ -281,10 +282,60 @@ <h2 class="title-post">
             </ul>
 
         </section>
+        <!-- Contact form -->
+        <section class="contact-form">
+            <span class="contact-intro-cont">
+                <p>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </p>
+            </span>
+            <form action="https://formspree.io/f/xlevajjz" method="POST" class="the-form">
+                <span class="name-div">
+                    <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required>
+                </span>
+                <span class="email-div">
+                    <input type="email" name="email" id="email" placeholder="Email address" required>
+                </span>
+                <span class="message-div">
+                    <textarea name="message" id="message" cols="42" rows="5" placeholder="Enter text here"
+                        maxlength="500" required></textarea>
+                </span>
+                <button type="submit" class="get-in-touch">
+                    <span>Get in touch</span>
+                </button>

```

Comment:
- [OPTIONAL] According to the requirements you should Implement the button interactions (enable, hover, etc.). 💯

![image](https://user-images.githubusercontent.com/45231276/200808025-02fea14a-e54c-4c5e-8ad8-4c2c6842792f.png)